.stylua.toml

```diff
@@ -1,3 +1,4 @@
+column_width = 300
+
 [sort_requires]
-enabled = true
-column_width = 300
\ No newline at end of file
+enabled = true
```

Comment:
did the trailing newline get deleted here?

Reply:
OK will change